float32 cputemp
float32 projectortemp